PLUGINS.md

```diff
@@ -92,30 +93,32 @@ my-plugin/
 ├── ...
 └── platforms/
     ├── android/
-    │   └── res/
-    │   └── MyLibrary.jar
-    │   └── MyLibrary.aar
-    │   └── include.gradle
+    │   ├── res/
+    │   ├── MyLibrary.jar
+    │   ├── MyLibrary.aar
+    │   ├── include.gradle
     │   └── AndroidManifest.xml
     └── ios/
-        ├── MyiOSLibrary.framework
+        ├── MyiOSFramework.framework
+        ├── build.xcconfig
+        ├── Podfile
         └── Info.plist
 ```
 
-* `platforms\android`: This directory contains any native Android libraries packaged as `*.jar` and `*.aar` packages. These native libraries can reside in the root of this directory or in a user-created sub-directory. During the plugin installation, the NativeScript CLI will configure the Android project in `platforms\android` to work with the plugin. 
+* `platforms\android`: This directory contains any native Android libraries packaged as `*.jar` and `*.aar` packages. These native libraries can reside in the root of this directory or in a user-created sub-directory. During the plugin installation, the NativeScript CLI will configure the Android project in `platforms\android` to work with the plugin.
 * `platforms\android\res`:  (Optional) This directory contains resources declared by the `AndroidManifest.xml` file. You can look at the folder structure [here](http://developer.android.com/guide/topics/resources/providing-resources.html#ResourceTypes).
-* `platforms\ios`: This directory contains native iOS frameworks (both static and shared). During the plugin installation, the NativeScript CLI will copy these files to `lib\iOS` in your project and will configure the iOS project in `platforms\ios` to work with the frameworks.
-* `platforms\android\include.gradle`: This file modifies the native Android configuration of your NativeScript project such as native dependencies, build types and configurations. For more information about the format of `include.gradle`, see [include.gradle file](#includegradle-specification).
+* `platforms\android\include.gradle`: This file modifies the native Android configuration of your NativeScript project such as native dependencies, build types and configurations. For more information about the format of `include.gradle`, see [`include.gradle` file](#includegradle-specification).
+* `platforms\ios`: This directory contains native iOS frameworks (both static and shared). During the plugin installation, the NativeScript CLI will copy these files to `lib\iOS` in your project and will configure the iOS project in `platforms\ios` to work with the frameworks. If the library is written in Swift, only APIs exposed to Objective-C are exposed to NativeScript.
```

Comment:
I've reordered these a bit. I believe they are consistent now. As I see it there are two examples - one with JS code only and the other with native dependencies.